### ISSUE TITLE
Stegcracker Not Finding Known Password

### ISSUE THREAD

**Issue by wallofsheep** [Paradoxis/StegCracker#9]
Did the following with the same issue noted in the original bug ticket.

Updated to the latest:
pip3 install stegcracker -U --force-reinstall

steghide embed -cf image.jpg -ef secret.txt
created a password of "password" and several other words in it for testing.

created a file named dict.txt with the word "password" in it.
ran:
stegcracker image.jpg dict.txt
Error: Failed to crack file, ran out of passwords.

Tried downloading tom.txt and using a password from that file with a image.jpg and I get the same error.

_Originally posted by @wallofsheep in https://github.com/Paradoxis/StegCracker/issues/4#issuecomment-476894231_

**Comment by Paradoxis** [comment]
Did you remove the previous installation of stegcracker? You can test this by running: 

    cat $(which stegcracker) 

It should return the contents of a Python script If the output is the latest verison, try running it with the --verbose option

**Comment by wallofsheep** [comment]
```
root@test:~/stegcracker_test# cat $(which stegcracker) 
#!/usr/bin/python3
# -*- coding: utf-8 -*-
import re
import sys

from stegcracker.__main__ import main

if __name__ == '__main__':
    sys.argv[0] = re.sub(r'(-script\.pyw?|\.exe)?$', '', sys.argv[0])
    sys.exit(main())
```

StegCracker - (https://github.com/Paradoxis/StegCracker)
Copyright (c) 2019 - Luke Paris (Paradoxis)

usage: stegcracker <file> [<wordlist>]
stegcracker: error: the following arguments are required: file

**Comment by Paradoxis** [comment]
Could you add the file & worslist you’re using? I’ll look into it after I wake up :) (Extra info like platform you’re using and the expected password etc is appreciated)

**Comment by wallofsheep** [comment]
Attached is image.jpg

The password is "th" - I used tom.txt from the example that worked (also attached).

Platform: Kali Linux (latest weekly distrib running in VMWare)

Thanks


[tom.txt](https://github.com/Paradoxis/StegCracker/files/3010970/tom.txt)


![image](https://user-images.githubusercontent.com/4331797/55041690-601e0600-4feb-11e9-9098-8b511f9681c7.jpg)

**Comment by Paradoxis** [comment]
Fixed your issue in commit cf3b8b5, passwords weren't being stripped when passed to the cracker, releasing it in a bit :)
